lib/haskell/natural4/src/LS/XPile/MathLang/GenericMathLang/GenericMathLangAST.hs

```diff
@@ -78,9 +78,9 @@ type Number = Double
 
 -- | GML type information can either stem from annotations or from inference.
 --
-data TLabel = FromUser L4EntType
```

Comment:
As the person who was responsible for this `TLabel` thing, I wonder if it might not be better to just do away with it, since it's probably not being used elsewhere. (Or if you think you might need something like it, feel free to change it as you like to suit your purposes.)

Reply:
Good to know! So far, I am not sure what we'll need, so I am not deleting it... for now :innocent: